cmd/pulumi-test-language/tests.go

```diff
@@ -1561,3 +1579,34 @@ func secret(x any) any {
 		"value":                            x,
 	}
 }
+
+type assertNoSecretLeaksOpts struct {
+	IgnoreResourceTypes []tokens.Type
+	Secrets             []string
+}
+
+func assertNoSecretLeaks(l require.TestingT, snap *deploy.Snapshot, opts assertNoSecretLeaksOpts) {
+	// Remove ConfigFetcher from snap to exclude it from secret leak checks.
```

Comment:
Should this be:

```suggestion
	// Remove certain types from the snap to exclude it from the secret leak checks.
```

?  We're only doing the `ConfigFetcher` in this PR, but this function can deal with others as well?